addons/mail/static/src/new/chat/chat_window_container.js

```diff
@@ -11,9 +11,27 @@ export class ChatWindowContainer extends Component {
 
     setup() {
         this.messaging = useMessaging();
+        this.state = useState({
+            isActive: false,
+        });
+        const updateIsActiveState = () => {
+            this.state.isActive = !this.messaging.state.discuss.isActive;
+        };
+        this.env.bus.addEventListener("ACTION_MANAGER:UI-UPDATED", updateIsActiveState);
+        onWillDestroy(() => {
+            this.env.bus.removeEventListener("ACTION_MANAGER:UI-UPDATED", updateIsActiveState);
+        });
+        this.messaging.isReady.then(() => {
+            if (
+                this.messaging.state.discuss.menuId !==
+                this.env.services.router.current.hash.menu_id
+            ) {
+                updateIsActiveState();
+            }
+        });
```

Comment:
You can use `onWillStart(this.messaging.isReady)`

Reply:
As we discussed, the flickering is still present but we'll take care of that later :+1: